doc/classes/RenderingServer.xml

```diff
@@ -3418,14 +3418,24 @@
 				Sets the screen-space roughness limiter parameters, such as whether it should be enabled and its thresholds. Equivalent to [member ProjectSettings.rendering/anti_aliasing/screen_space_roughness_limiter/enabled], [member ProjectSettings.rendering/anti_aliasing/screen_space_roughness_limiter/amount] and [member ProjectSettings.rendering/anti_aliasing/screen_space_roughness_limiter/limit].
 			</description>
 		</method>
-		<method name="set_boot_image">
+		<method name="set_boot_image" deprecated="This method is no longer in use since Godot 4.5.">
```

Comment:
The deprecation message should mention the replacement.
Remove `set_boot_image_with_stretch` link from the description and add it here instead.